### ISSUE TITLE
Build Failure: 'scaleBy' Method Not Defined in 'Point<double>' Class

### ISSUE THREAD

**Issue by lapin7771n** [tlserver/flutter_map_location_marker#120]
**Describe the bug**
When attempting to launch the Flutter application on an iPhone 15 Pro Max in debug mode, the build fails with an error related to the `flutter_map_location_marker` package. Specifically, the method `scaleBy` isn't defined for the class `Point<double>`.

**To Reproduce**
1. Go to the project directory.
2. Run the command flutter run to launch the app.
3. Select the iPhone simulator/device.
4. Observe the build process and see the error.

**Expected behavior**
The application should build successfully and launch on the iPhone simulator without encountering any errors related to the 'flutter_map_location_marker' package.

**Desktop (please complete the following information):**
 - OS: [e.g. macOS Sonoma Version 14.5 (23F79)]
 - Browser: N/A
 - Version: N/A

**Smartphone (please complete the following information):**
 - Device: iPhone 15 Pro Max
 - OS: iOS 17.5
 - Browser: N/A
 - Version: N/A

**Additional context**
The error log is as follows:

```vbnet
Launching lib/main.dart on iPhone 15 Pro Max in debug mode...
Running Xcode build...
Xcode build done.                                           20.9s
Failed to build iOS app
Error (Xcode): ../../../.pub-cache/hosted/pub.dev/flutter_map_location_marker-8.0.8/lib/src/options/focal_point.dart:46:20: Error: The method 'scaleBy' isn't defined for the class 'Point<double>'.
```

**Comment by olavurs** [comment]
Same issue here

**Comment by miakh** [comment]
same for method substract

**Comment by miakh** [comment]
#121

**Comment by Jeius** [comment]
Same issue here, I tried dependency overrides to fix the dependency issue and when I use 'flutter build web --release --wasm', I get  Error: The method 'scaleBy' isn't defined for the class 'Point<double>'.
 - 'Point' is from 'dart:math'.
      (size * 0.5).scaleBy(ratio) + offset;

**Comment by tlserver** [comment]
Fixed in V9.0.0
